scenarios/multi_object_tracking/01_training_introduction.ipynb

```diff
@@ -0,0 +1,820 @@
+{
```

Comment:
<ul><li>For mvp this is good. For the final version, we'll want to change this to the same XML syntax used for object detection. The only extra information in your dataset is the re-id value.</li><li>Does FairMOT only support a single class? If so, we could build a dedicated "can" dataset using simply a subset of the images we have already with only cans being annotated. </li><li>As for "<code>FridgeObjects.train", </code>that is FairMOT specific and probably not something we want to expose to the user.</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/545/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>